packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -1,90 +1,117 @@
 /*
- * Copyright (c) 2020, salesforce.com, inc.
+ * Copyright (c) 2019, salesforce.com, inc.
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
 import {
+  Config,
   ConfigAggregator,
-  ConfigFile,
-  ConfigValue,
   OrgConfigProperties,
+  SfConfigProperties,
   StateAggregator
 } from '@salesforce/core';
-import * as path from 'path';
 import { ConfigAggregatorProvider } from '../providers';
-import { TelemetryService } from '../telemetry/telemetry';
-import { getRootWorkspacePath } from '../workspaces';
 
 export enum ConfigSource {
   Local,
   Global,
   None
 }
 
```

Comment:
This was basically combining the new configUtils in core with the new configUtils in vscode-utils.